system/netdata.logrotate.in

```diff
@@ -0,0 +1,15 @@
+@localstatedir_POST@/log/netdata/*.log {
+	daily
+	missingok
+	rotate 14
+	compress
+	delaycompress
+	notifempty
+	sharedscripts
+	postrotate
+		if service netdata status > /dev/null ; then \
+			service netdata restart > /dev/null; \
+		fi;
```

Comment:
I think netdata needs a `reload` option (normally triggered via SIGHUP) which only reopens the log files but does not restart any processes. copytruncate is an ugly hack which normally should be avoided unless the software really can't be changes to behave correctly (i.e. most enterprise Java solutions).